src/App.js

```diff
@@ -1,13 +1,68 @@
 import React from 'react';
+import { GoodsList } from './components/GoodsList/GoodsList';
 
 import './App.scss';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+import { getAll, get5First, getRed } from './api/goods';
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+class App extends React.Component {
+  state = {
+    goods: '',
+  }
+
+  loadAllGoods = () => {
+    getAll()
+      .then((goods) => {
+        this.setState({
+          goods,
+        });
+      });
+  }
+
+  loadFirst5 = () => {
+    get5First()
+      .then((goods) => {
+        this.setState({
+          goods,
+        });
+      });
+  }
+
+  loadRedGoods = () => {
+    getRed()
+      .then(goods => this.setState({
+        goods,
+      }));
+  }
```

Comment:
your functions loadAllGoods, loadFirst5, loadRedGoods do the same things and you could remove them and use one which takes a needed function as a parameter